client/src/shared/components/SearchBar/SearchBar.tsx

```diff
@@ -0,0 +1,41 @@
+import { ChangeEvent, FormEvent, useState } from "react";
+
```

Comment:
- demonstrate the component by showing it in `/components` URL.
- add a file `./pages/shared/Components` put here all the components made.

`// in components url. You should display the following`
![image](https://user-images.githubusercontent.com/73863685/222058717-56283862-a700-48b0-971c-278239bf6f03.png)